### PR TITLE
remove xref_tigrfam field

### PR DESCRIPTION
**Pull request by austinhpatton**
This is a little hotfix for `v1.0.0-alpha` that removes the `xref_tigrfam` field from the uniprot query. Seems they've gotten rid of this without an announcement, or it otherwise is not queryable with bioservices. Regardless, it's got to go. 

Once we merge this, we make a shiny new release - `v1.0.1-alpha`

### CODE REVIEW

**Review by mertcelebi — COMMENTED** [review]
Can you also update the changelog with the new version and the change?

**Comment by austinhpatton** [comment]
Done! Let me know if you think the description is sufficient. I'm also going to update the name/description of the v1.0.0-alpha release to be more consistent.

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,4 +5,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## v1.0.0-alpha - 09/27/2023
 
-Initial release of Arcadia-Science/noveltree, created with the [nf-core](https://nf-co.re/) template.
+Initial release of NovelTree.
+
+## v1.0.1-alpha - 09/28/2023
```
Comment (mertcelebi) [inline CHANGELOG.md:10]: Can you add this to the top. So the order is descending by date. So the most recent change is the easiest to find

Comment (austinhpatton) [reply CHANGELOG.md:10]: Ah, yup! Makes sense - wasn't sure what the best/standard convention was

**Pull request merged**